deny.toml

```diff
@@ -157,4 +157,4 @@ skip = [
 
 # github.com organizations to allow git sources for
 [sources.allow-org]
-github = ["pcwalton", "servo", "gfx-rs"]
+github = ["pcwalton", "servo", "gfx-rs", "jdm"]
```

Comment:
I think this will cause an error once you start using servo/mozjs, I generally just keep this as is when using patched version of deps and let lint fail, as this provides a safeguard to prevent accidental landing a PR that is using one's fork.